### DISCUSSION TITLE
Onionsite Not Found in Tor Browser

### DISCUSSION THREAD

**Discussion by erwindegrave** [globaleaks/globaleaks-whistleblowing-software#3465] [General]
Hi,

I do have an issue with the Tor Browser. When I copy the onion link in the tro browser, I get a message 'Onionsite Not Found'.
![Screenshot 2023-05-30 at 17 51 20](https://github.com/globaleaks/GlobaLeaks/assets/19929116/53e20cb7-bb8e-439e-9df9-d97947903005)
Here is one of the links: sge52vmxrk34fez6rv6zy6ckljhc5nhbvceelnzaibqwyn2grx6orryd.onion

I have even regenerated the onion link, but that did not help.

Anyone has an idea how to solve this?

I have the newest software 4.11.5 installed.

Thanks

**Comment by evilaliv3** [comment]
Thank you for your question @erwindegrave 

The onion key and its address are generated by the application so they always work.

What is probably preventing your platform to be publicly reachable on toe could be to a firewall rule. Please check that you are not blocking outgoing connections or that your provider is not implementing specific rules to block Tor traffic.

**Comment by bsdzip** [comment]
I've got the same problem, seems tor doesn't create the control socket.

globaleaks.log
[-] [E] Failed to initialize Tor connection; error: Tor control port not open on /var/run/tor/control; waiting for Tor to become available

 ls -l /var/run/tor/
total 8
-rw-r----- 1 debian-tor debian-tor 32 Jun  2 09:54 control.authcookie
-rw-r--r-- 1 debian-tor debian-tor  4 Jun  2 09:54 tor.pid

Ubuntu 20.04.6 LTS

GlobaLeaks version: 4.11.5
Database version: 64

Tor version 0.4.2.7

Still say it's available though:
[-] GlobaLeaks is now running and accessible at the following urls:
2023-06-02 09:54:44+0200 [-] - [HTTP]   --> http://xyz
2023-06-02 09:54:44+0200 [-] - [HTTPS]  --> https://xyz
2023-06-02 09:54:44+0200 [-] - [Tor]:   --> http://zyx.onion
2023-06-02 09:54:44+0200 [-] Starting factory _HTTP11ClientFactory(<function HTTPConnectionPool._newConnection.<locals>.quiescentCallback at 0x7f7e1d1fd670>, <twisted.internet.endpoints._WrapperEndpoint object at 0x7f7e1d1e4520>)
2023-06-02 09:54:45+0200 [-] Stopping factory _HTTP11ClientFactory(<function HTTPConnectionPool._newConnection.<locals>.quiescentCallback at 0x7f7e1d1fd670>, <twisted.internet.endpoints._WrapperEndpoint object at 0x7f7e1d1e4520>)

**Comment by evilaliv3** [comment]
Thank you for reporting this.

> Message ID: <globaleaks/GlobaLeaks/repo-discussions/3465/comments/6069281@
> github.com>
>

This should be investigated probably looking at the default configuration
of Tor on the latest Ubuntu.

Is it it possible for you to look into this and support this review?

**Reply by bsdzip** [reply]
A first glimpse at the configuration:

cat /var/run/tor/tor.pid
719
 ps -aux |grep 719
debian-+     719  0.0  2.7  55672 25728 ?        Ss   09:54   0:00 /usr/bin/tor --defaults-torrc /usr/share/tor/tor-service-defaults-torrc -f /etc/tor/torrc --RunAsDaemon 0

cat /usr/share/tor/tor-service-defaults-torrc
DataDirectory /var/lib/tor
PidFile /run/tor/tor.pid
RunAsDaemon 1
User debian-tor

ControlSocket /run/tor/control GroupWritable RelaxDirModeCheck
ControlSocketsGroupWritable 1
SocksPort unix:/run/tor/socks WorldWritable
SocksPort 9050

CookieAuthentication 1
CookieAuthFileGroupReadable 1
CookieAuthFile /run/tor/control.authcookie

Log notice syslog

No uncommented entries in torrc
cat /etc/tor/torrc |grep -v "^#"

cat /etc/tor/torsocks.conf |grep -v "^#"
TorAddress 127.0.0.1
TorPort 9050
OnionAddrRange 127.42.42.0/24

Permissions at tor run folder
ls -ld /var/run/tor
drwxr-sr-x 2 debian-tor debian-tor 80 Jun  2 09:54 /var/run/tor

**Comment by erwindegrave** [comment]
@evilaliv3 @bsdzip 
Do you have any update about this?
Thanks

**Comment by evilaliv3** [comment]
No actually. Which distribution are you using? Are you running any virtualization technology? (e.g. docker)

**Comment by erwindegrave** [comment]
I run version 4.5.11 and just running on a vm without virtualization.
